### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by wmdhosting** [wbrowar/craft-3-guide#50] [Ideas and Feature Requests]
I think this would be great way to show.. Guide 

![Snag_96f94cc3](https://user-images.githubusercontent.com/76176291/133213824-2063bcf1-a394-435d-abbd-2ff78b0b7f0e.png)

**Comment by wmdhosting** [comment]
![Snag_9709ee74](https://user-images.githubusercontent.com/76176291/133216571-1de0bb0d-7721-4cc3-a10f-8b9cbd6814c1.png)

Is it posiblle to do this to work as plugin update info. instead. this show hide buttons.

**Comment by wmdhosting** [comment]
![Snag_970dd78d](https://user-images.githubusercontent.com/76176291/133217251-7c0b99d4-8c52-4f6d-a144-ba0d3db9a127.png)
bug perhaps. 

only first created widget shows name.. 

So its not easy.. to reorder..

**Comment by wbrowar** [comment]
Thanks for these ideas, too, @wmdhosting! I'm going to move these to a discussion, under Ideas.

**Comment by wbrowar** [comment]
All of these should now be included in the changes in Guide 5 (except the widget naming suggestion). I will look into that some more, but I'm going to close this discussion for now. Thanks again for your suggestions!
